.travis.yml

```diff
@@ -35,23 +40,30 @@ before_install:
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
 
 install:
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION atlas numpy scipy matplotlib sympy cython $MAYAVI pytables pyparsing
-  - source activate test-environment
+
+  - conda create -q -n sfepy-test python=$TRAVIS_PYTHON_VERSION numpy scipy cython matplotlib pytables pyparsing sympy atlas
+  - source activate sfepy-test
+
+  - conda info -a
   - python -V
+
   - pip install scikit-umfpack
   - pip install https://bitbucket.org/dalcinl/igakit/get/default.tar.gz
 
 script:
+
   - python setup.py build_ext --inplace
   - python run_tests.py; export SFEPY_TEST_RESULTS=$?; (exit $SFEPY_TEST_RESULTS)
-  - |
-    if [ $SFEPY_TEST_RESULTS -ne 0 ]; then
-      python run_tests.py --raise
-    fi
+
+#
+# Probably not needed here...
+#
+#  - |
+#    if [ $SFEPY_TEST_RESULTS -ne 0 ]; then
+#      python run_tests.py --raise
+#    fi
```

Comment:
This is useful, because "python run_tests.py" does not provide any useful info on why a test failed. That is why on failure we re-run the tests without the silenced output. Maybe `--filter-none` instead of `--raise` might be better so that it does not stop on the first error.

Another possibility is to add `--filter-none` to the first call.

Reply:
OK, I'll add --filter-none switch to first call...

P.S. My original idea was: you get pass/fail info on each test and you can re-run test localy.

Reply:
Well, you may not be able to run it locally, if you do not have that particular platform ;)

Actually, `--raise` is better - `--filter-none` does not show the exception. So I guess the original version is the most convenient.